### ISSUE TITLE
Issue with big images >1,5 MB

### ISSUE THREAD

**Issue by Agrofatz** [axelerant/typo3-importer#7]
When importing articles with big images (2-4 images in the article and one of them bigger than ca. 1, MB), the import will not work. Means: The text will be imported, but not the picture (and no descriptions etc. for the image)

Error: function (){return f&&(c&&!b&&(h=f.length-1,g.push(c)),function d(b){n.each(b,function(b,c){n.isFunction(c)?a.unique&&j.has(c)||f.push(c):c&&c.length&&”string”!==n.type(c)&&d(c)})}(arguments),c&&!b&&i()),this} 

Also if I put the settings to “no related media to import” it works well.
I already put the maximum memory-limit to 600 MB and added the SetEnv MAGICK_THREAD_LIMIT 1 to my htaccess, but still every import with more than one picture failes.

**Comment by Agrofatz** [comment]
During my searching I found some articles, where the image wasnt to big....

Maybe these URLs will help:
Import failed:
http://www.kabinett-online.de/kultur/artikel/datum/2015/12/04/vorbildliche-bauten-2015-in-nrw-ausgezeichnet.html
http://www.kabinett-online.de/kultur/artikel/datum/2016/01/18/in-inniger-verbundenheit.html
http://www.kabinett-online.de/kultur/artikel/datum/2015/10/29/cool-columbus-cube-continium.html
http://www.kabinett-online.de/politik/artikel/datum/2015/12/23/papst-franziskus-wird-karlspreistraeger-2016.html
http://www.kabinett-online.de/aktuelle-themen/artikel/datum/2016/01/21/die-lage-ist-ernst-in-deutschland-und-in-europa.html

Import success:
http://www.kabinett-online.de/staedte/bonn/artikel/datum/2015/02/05/wolfgang-fuchs-zum-stadtdirektor-gewaehlt.html
http://www.kabinett-online.de/staedte/bonn/artikel/datum/2016/04/06/lasst-uns-gemeinsam-bruecken-bauen.html

**Comment by michael-cannon** [comment]
@Agrofatz thank you for letting us know about this image size issue.

@saurabhsirdixit are you able to get in a quick look by this time next week?

**Comment by saurabhsirdixit** [comment]
@michael-cannon Yes, I requested him to create an issue in Git so that we can work on it and he can track. 
@Agrofatz Thanks for the detailed issue. I am looking forward to getting a fix and will update here. Thanks.

**Comment by Agrofatz** [comment]
Hi there,

any ideas so far to solve my problem?

Regards

Sascha


Am 24.05.2017 um 12:21 schrieb Saurabh Dixit:

**Comment by saurabhsirdixit** [comment]
@Agrofatz I was just going to update you on this. I need two more days to reproduce this and then a fix. I will let you know and look forward to sending you RC version.
Is it possible for you to point me to your dev server from where I can directly reproduce it?

**Comment by Agrofatz** [comment]
Hi there, yes I can help you with my dev server. what do you need?


Am 06.06.2017 um 13:52 schrieb Saurabh Dixit:

**Comment by saurabhsirdixit** [comment]
@Agrofatz We need your dev server website address with admin credential to access the backend. 

Further, in case I'm updating software directly on your website, please provide me with SSH/FTP login credentials for your website.

**Comment by Agrofatz** [comment]
OK, so a WordPress user and FTP access?


Am 06.06.2017 um 13:59 schrieb Saurabh Dixit:

**Comment by saurabhsirdixit** [comment]
Exactly. Thanks :)

**Comment by Agrofatz** [comment]
OK, here u go: (its all in german...)

FTP:
URL: p400812.webspaceconfig.de
p400812
helpMe17?

WP:
URL: http://wordpress.p400812.webspaceconfig.de/wp-admin
Github
Twb!-17U

If you need any help, let me know!

One last question: If you can fix it, so it will run without errors. Is 
it possible to clean the database and delete all the old imports (inkl. 
images) to start then new?
WIll Wordpress delete all media too?


Am 06.06.2017 um 14:02 schrieb Saurabh Dixit:

**Comment by saurabhsirdixit** [comment]
I will check and update you on this. 

Thanks for the credentials. I will update you soon.

**Comment by subharanjanm** [comment]
@Agrofatz @saurabhsirdixit Can we please delete the above credentials and not provide the server login details in public? @Agrofatz , You should send the login details through email to @saurabhsirdixit . Thanks !!

**Comment by saurabhsirdixit** [comment]
Thanks @subharanjan. I don't see an option to delete that message. Can you check once?
If not then @agrofatz please change the above shared credential and send us new credential to our email id: support@axelerant.com.

Thanks

**Comment by Agrofatz** [comment]
Ok, i deleted the comment. New credentials follow to the new email.


Am 06.06.2017 um 20:01 schrieb Saurabh Dixit:
